testsuite/manualmode/src/test/java/org/wildfly/core/test/standalone/mgmt/HTTPSConnectionWithCLITestCase.java

```diff
@@ -75,7 +74,6 @@
 
 @RunWith(WildflyTestRunner.class)
 @ServerControl(manual = true)
-@Ignore("[WFCORE-1954] Wrap CLIENT_CERT behaviour of legacy realms with Elytron components.")
```

Comment:
Testing this on master-ignore with a number of other changes, this test failed on windows:

https://ci.wildfly.org/viewLog.html?buildId=33100&tab=buildResultsDiv&buildTypeId=WildFlyCore_MasterIgnorWindows

The log didn't show anything. The test does a number of config mods to set up HTTPS on the management interface and then adds a native interface so the CLI can connect and do a reload. The log shows a message that the server is listening on 9999 but then when the CLI tries to connect to 9999 it fails.